assembl/static/js/app/views/attachments.js

```diff
@@ -183,12 +189,13 @@ var AttachmentEditableView = AbstractAttachmentView.extend({
   _renderAttachmentPurpose: function(){
     var purposesHtml = [],
         that = this;
-    _.each(Attachments.attachmentPurposeTypes, function(attachmentPurposeDef) {
-      purposesHtml.push('<li><a class="js_attachmentPurposeDropdownListItem" data-id="' + attachmentPurposeDef.id + '" data-toggle="tooltip" title="" data-placement="left" data-original-title="' + attachmentPurposeDef.id + '">' + attachmentPurposeDef.label + '</a></li>');
-    });
-
+    if(this.model.get('@type') !== 'IdeaAttachment'){
```

Comment:
I'm not very comfortable with this. To recap: To me the view of the image attached to the idea panel and its UX are so different from the "regular" attachment view which is used in a message (and which may be used in other places of Assembl where we could upload a list of files), that it needs its own view (which could inherit from a common parent). I guess this line of code which displays things differently depending on the type of the object which uses the view should be avoided in general, and this is an epiphenomenon of the problem. Others, any opinion?